compiler/rustc_metadata/src/rmeta/decoder/cstore_impl.rs

```diff
@@ -496,6 +496,13 @@ impl CStore {
     ) -> impl Iterator<Item = (DefId, DefId, Option<SimplifiedType>)> + '_ {
         self.get_crate_data(cnum).get_trait_impls()
     }
+
+    pub fn inherent_impls_in_crate_untracked(
```

Comment:
Please add a doc comment about what the returned DefIds are (maybe to `trait_impls_in_crate_untracked` above too).